stc_game/Assets/Scripts/HUD/PotionCooldown.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PotionCooldown : MonoBehaviour
+{
+    [SerializeField] public Transform parent;
+    private float cooldown;
+    public Image image;
+    public float startTime;
+
+    private void Awake()
+    {
+        cooldown = parent.GetComponent<PotionButton>().cooldown;

```

Comment:
Is there a way to get these by Id? down the road those won't be scalable to only reference by type

Reply:
I'm not sure. This just grabs the value from the parent gameobject in the hierarchy. And the only way to set the value is in the editor for now. I just wanted to get a prototype of these potions to work.